packages/@expo/cli/src/start/platforms/android/emulator.ts

```diff
@@ -23,16 +23,23 @@ export function whichEmulator(): string {
 export async function listAvdsAsync(): Promise<Device[]> {
   try {
     const { stdout } = await spawnAsync(whichEmulator(), ['-list-avds']);
-    return stdout
-      .split(os.EOL)
-      .filter(Boolean)
-      .map((name) => ({
-        name,
-        type: 'emulator',
-        // unsure from this
-        isBooted: false,
-        isAuthorized: true,
-      }));
+    return (
+      stdout
+        .split(os.EOL)
+        .filter(Boolean)
+        /**
+         * AVD names cannot contain spaces. This removes extra info lines from the output. e.g.
+         * "INFO    | Storing crashdata in: /tmp/android-brent/emu-crash-34.1.18.db
+         */
+        .filter((name) => !name.trim().includes(' '))
```

Comment:
I don't think this is true, [from Google's AVD docs](https://developer.android.com/studio/run/managing-avds#avdproperties)

<img width="853" alt="image" src="https://github.com/expo/expo/assets/1203991/80047aa2-33f7-46ff-a7c9-750e8440d8d2">

Reply:
oh, I didn't mean that type of "name", I guess the correct term would be "AVD ID". Even if you rename an AVD through Android studio the returned id when running `emulator -list-avds`  will never include spaces. let me update the comment

Reply:
updated the comment

Reply:
Gotcha, can confirm:

<img width="415" alt="image" src="https://github.com/expo/expo/assets/1203991/f9df4323-9fa8-437e-a0c5-cadec6b25453">